src/pages/workspace/accounting/qbo/import/QuickbooksLocationsPage.tsx

```diff
@@ -21,6 +21,8 @@ function QuickbooksLocationsPage({policy}: WithPolicyProps) {
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '';
     const {syncLocations, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const shouldBeDisabled =
+        !syncLocations && policy?.connections?.quickbooksOnline?.config.reimbursableExpensesExportDestination !== CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY;
```

Comment:
NAB but the code would be neater if we define a `reimbursableExpensesExportDestination` variable  in `const {syncLocations, pendingFields,reimbursableExpensesExportDestination}` and use it here. 
Also, can you please update the branch with the latest main?
Hoping to complete the checklist today

Reply:
good point! I didn't see we're destructuring above 😄